frontend-opal/src/components/PortfolioContainer/News/News.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { SafeAreaView, Text, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Card, CardItem, Spinner } from 'native-base';
+import GetLatestNews from './GetLatestNews';
+import NewsItems from './NewsItems';
+import styles from './styles';
+
+const News = ({ portfolio }) => {
+  const displayedNews = GetLatestNews(portfolio);
```

Comment:
@WenyuD this should be very similar to the other portfolio elements
NewsContainer -> from connected redux portfolio produces the news and connects the presenter with the news
NewsPresenter -> just have the news as prop